packages/mysql/src/Adaptor.js

```diff
@@ -19,273 +87,176 @@ import squel from 'squel';
  */
 
 export function sql(sqlQuery, options = {}) {
```

Comment:
Can we update the main docs to say a bit more about values? There's already some stuff about injection attacks - let's just make sure it's consistent